Error on archived repositories

Hello @mbestavros and thank you for sharing this nice action with the world.

I used your action for a while without any problems, until I archived one of my repositories. Now the actions runs fine until it *hits* the archived repo.

```log
 File "/home/runner/work/_actions/actions-automation/manage-your-labels/main/src/mirror-labels", line 177, in <module>
    githubgql.graphql(CREATE_LABEL, token=token, accept="application/vnd.github.bane-preview+json", input={
  File "/home/runner/.local/lib/python3.10/site-packages/githubgql/githubgql.py", line 125, in graphql
    raise GraphQLError(data["errors"])
githubgql.githubgql.GraphQLError: [{'type': 'UNPROCESSABLE', 'path': ['createLabel'], 'locations': [{'line': 2, 'column': 5}], 'message': 'Repository was archived so is read-only'}]
Error: Process completed with exit code 1.
```
https://github.com/figuren-theater/.github/actions/runs/5424170871/jobs/9863140119

Is there anything I can do, to prevent this error? 
Would be great to read your opinion on this.

--- a/src/mirror-labels
+++ b/src/mirror-labels
@@ -10,7 +10,7 @@ import yaml
 QUERY = '''
 query($owner:String!, $reposCursor:String, $labelsCursor:String) {
     organization(login:$owner) {
-        repositories(first:100, after:$reposCursor) {
+        repositories(first:100, after:$reposCursor, isArchived:false) {
             pageInfo { endCursor hasNextPage }
             nodes {
                 name

